apisix/plugins/kafka-logger.lua

```diff
@@ -51,6 +51,16 @@ local schema = {
                 },
             },
         },
+        sasl_config = {
+            type = "object",
+            description = "sasl config",
+            properties = {
+                mechanism = { type = "string", description = "mechanism", default = "PLAIN" },
```

Comment:
The `mechanism` should be restricted to only use a few enumerations like `PLAIN`, `SCRAM` and so on.

Reply:
OK, `SCRAM-SHA-256` is copied from Confluent docs. No worries at all.